project-base/phpstan.neon

```diff
@@ -25,7 +25,7 @@ parameters:
         -
             # Ignore annotations in generated code
             message: '#^PHPDoc tag @param for parameter \$function with type callable is not subtype of native type Closure.$#'
-            path: %currentWorkingDirectory%/project-base/tests/App/Test/Codeception/_generated/AcceptanceTesterActions.php
```

Comment:
Does it make sense to have this change and split of install script in upgrade notes?

Reply:
yes, it does, after our discussion, I rebased it to `9.1` branch + I will add the upgrade notes in a fixup commit(s) :+1:

Reply:
fixed in f16bd554d and 82134dbc5